.prettierignore

```diff
@@ -1,5 +1,7 @@
 node_modules
 artifacts
 cache
+typechain
 coverage*
 gasReporterOutput.json
+.*
```

Comment:
why ignore all files ?

Reply:
used before the use of eslint to prettier ts/js files. fixed